New package: FirebirdProject.Firebird.4 version 4.0.3.2975

Dragon1573 opened the pull request:
## What's changed

- Resolve https://github.com/microsoft/winget-pkgs/issues/119010

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122170)

Dragon1573 committed:
New package: FirebirdProject.Firebird.4 version 4.0.3.2975

mdanish-kh commented on manifests/f/FirebirdProject/Firebird/4/4.0.3.2975/FirebirdProject.Firebird.4.installer.yaml:
```diff
@@ -0,0 +1,37 @@
+  InstallerUrl: https://github.com/FirebirdSQL/firebird/releases/download/v4.0.3/Firebird-4.0.3.2975-0-x64.exe
+  InstallerSha256: DB2091AC5E1C232366B37946ECC3B7DF751BC98669791389C49CD137E9C8EBC1
+  AppsAndFeaturesEntries:
+  - DisplayName: Firebird 4.0.3.2975 (x64)
+    Publisher: Firebird Project
+    DisplayVersion: 4.0.3.2975
+    ProductCode: FBDBServer_4_0_x64_is1
```
```suggestion
  - DisplayName: Firebird 4.0.3.2975 (x64)
    Publisher: Firebird Project
    ProductCode: FBDBServer_4_0_x64_is1
```
DisplayVersion should not be used when it's equal to PackageVersion. See https://github.com/microsoft/winget-pkgs/pull/65816#issue-1301301147

mdanish-kh commented on manifests/f/FirebirdProject/Firebird/4/4.0.3.2975/FirebirdProject.Firebird.4.installer.yaml:
```diff
@@ -0,0 +1,37 @@
+- Architecture: x86
+  InstallerUrl: https://github.com/FirebirdSQL/firebird/releases/download/v4.0.3/Firebird-4.0.3.2975-0-Win32.exe
+  InstallerSha256: 125BB94043973CB7DCB84B0F1673D03747E9769EB0CDA0F64227F0974A323182
+  AppsAndFeaturesEntries:
+  - DisplayName: Firebird 4.0.3.2975 (Win32)
+    Publisher: Firebird Project
+    DisplayVersion: 4.0.3.2975
+    ProductCode: FBDBServer_4_0_is1
```
```suggestion
  AppsAndFeaturesEntries:
  - DisplayName: Firebird 4.0.3.2975 (Win32)
    Publisher: Firebird Project
    ProductCode: FBDBServer_4_0_is1
```

DisplayVersion should not be used when it's equal to PackageVersion. See https://github.com/microsoft/winget-pkgs/pull/65816#issue-1301301147

mdanish-kh commented on manifests/f/FirebirdProject/Firebird/4/4.0.3.2975/FirebirdProject.Firebird.4.installer.yaml:
```diff
@@ -0,0 +1,37 @@
+    Publisher: Firebird Project
+    DisplayVersion: 4.0.3.2975
+    ProductCode: FBDBServer_4_0_x64_is1
+  - DisplayName: MSI to redistribute MS VS2017 CRT libraries for x64
+    Publisher: The Firebird Project
+    DisplayVersion: 14.16.27012.6
+    ProductCode: '{988BFDD3-1E41-46D7-AD29-C805348B0F59}'
+  - DisplayName: MSI to redistribute MS VS2017 CRT libraries for x86
+    Publisher: The Firebird Project
+    DisplayVersion: 14.16.27012.6
+    ProductCode: '{9B0B1AD9-440A-4E79-A019-1DEBD2718DA4}'
```
```suggestion
```
Only primary ARP entry should be included in the manifest

mdanish-kh commented on manifests/f/FirebirdProject/Firebird/4/4.0.3.2975/FirebirdProject.Firebird.4.installer.yaml:
```diff
@@ -0,0 +1,37 @@
+    Publisher: Firebird Project
+    DisplayVersion: 4.0.3.2975
+    ProductCode: FBDBServer_4_0_is1
+  - DisplayName: MSI to redistribute MS VS2017 CRT libraries for x86
+    Publisher: The Firebird Project
+    DisplayVersion: 14.16.27012.6
+    ProductCode: '{9B0B1AD9-440A-4E79-A019-1DEBD2718DA4}'
```
```suggestion
```
Only primary ARP entry should be included in the manifest

mdanish-kh commented on manifests/f/FirebirdProject/Firebird/4/4.0.3.2975/FirebirdProject.Firebird.4.locale.en-US.yaml:
```diff
@@ -0,0 +1,27 @@
+PackageName: Firebird
+PackageUrl: https://firebirdsql.org/en/firebird-4-0/
+License: Initial Developer's Public License Version 1.0
+LicenseUrl: https://firebirdsql.org/en/initial-developer-s-public-license-version-1-0/
```
```suggestion
LicenseUrl: https://github.com/FirebirdSQL/firebird/#licensing
```

This URL provides the user with more context about licensing

Dragon1573 committed:
Fixup: Remove redundant ARP Entries

- Only primary ARP entry should be included in the manifest.

Dragon1573 committed:
Fixup: Change LicenseUrl

- This URL provides the user with more context about licensing.

Dragon1573 committed:
Fixup: Remove "DisplayVersion" in ARP Entries

- DisplayVersion should not be used when it's equal to PackageVersion.
- See https://github.com/microsoft/winget-pkgs/pull/65816#issue-1301301147
